src/lib.rs

```diff
@@ -38,17 +38,20 @@ fn encode(current: f64, previous: f64, factor: i32) -> Result<String, String> {
     Ok(output)
 }
 
-/// Encodes a Google Encoded Polyline.
+/// Encodes a Google Encoded Polyline. Accepts a Vec or slice
```

Comment:
How about "Accepts a slice or anything (such as a vec) that can deref to a slice"? I'm just thinking of making it explicit for people who aren't necessarily aware of the exact relationship between the two.